fvm/src/gas/price_list.rs

```diff
@@ -244,6 +357,17 @@ pub struct PriceList {
     pub(crate) verify_post_discount: bool,
     pub(crate) verify_consensus_fault: i64,
     pub(crate) verify_replica_update: i64,
+    // 1 Fuel = gas_per_fuel * 1 Gas
+    pub(crate) gas_per_fuel: i64,
+
+    pub(crate) extern_traversal_cost: i64,
+    // TODO: Some of these will probably need per-byte multipliers
```

Comment:
I would introduce two variables here:

- `block_memcpy_per_byte_cost`
- `block_io_per_byte_cost`

These two factors composed appropriately at the syscall site should be enough to apply the appropriate dynamic cost portion per syscall.

Reply:
This is gonna get hard to shoehorn to fit both v15 and v16 -- in v15 we only have the per-byte cost for link. If we replace `block_link_per_byte` with `block_io_per_byte`, and start using `block_io_per_byte` for operations like `block_open`, we'll break consensus with v15.

I'm gonna leave `block_link_per_byte` in for now to get around this, but otherwise incorporate your suggestions.